Task1/Fridge.cs

```diff
@@ -0,0 +1,81 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task1
+{
+    public delegate void FridgeDelegate(object sender, FridgeEventArgs e);
+
+    public class Fridge
```

Comment:
Current logic does not notifies about what action was performed. You pass to subscriber only current state.

Reply:
-2 to logic.